interpreter/package_test.go

```diff
@@ -365,7 +365,6 @@ func TestInterpreter_EvalPackage(t *testing.T) {
 */
 
 func TestInterpreter_SetNewOption(t *testing.T) {
```

Comment:
Is this test testing anything anymore? It seems to be testing whether we can add an option to an existing package. This isn't possible to do so I'm not sure what this is testing.

Reply:
What's the correct behavior? I thought that the interpreter did not mutate packages anymore. Is this the case? If so, then either this test should be checking to see if the option has been set in the scope, or we should just remove the test as we already have tests in place for option assignments.

Reply:
The correct behavior is that setting an option will modify the value in the original package. So that way this would work as this is my understanding of options.

```
package influxdb

option token = ""

builtin _from
from = (bucket, token=token) => _from(bucket: bucket, token: token)

package main

import "influxdata/influxdb"
option influxdb.token = "abc"

influxdb.from(bucket: "telegraf") |> ...
```

The option would modify the value in the `influxdata/influxdb` scope as if it were a pointer to that value. What isn't possible is doing `option influxdb.token2 = "blahblah"` because the option wasn't defined in the original scope. This is allowed even if it is a no-op just because the previous behavior allowed it.

Reply:
@jsternberg I have updated the test accordingly.